x-pack/solutions/observability/plugins/apm/server/lib/helpers/get_esql_client.ts

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import {
+  type ObservabilityElasticsearchClient,
+  createObservabilityEsClient,
+} from '@kbn/observability-utils-server/es/client/create_observability_es_client';
+import type { APMIndices } from '@kbn/apm-data-access-plugin/server';
+import { APM_SERVER_FEATURE_ID } from '../../../common/rules/apm_rule_types';
+import type { MinimalAPMRouteHandlerResources } from '../../routes/apm_routes/register_apm_server_routes';
+
+export interface EsClient extends ObservabilityElasticsearchClient {
+  indices: APMIndices;
+}
+
+export async function getEsClient({
```

Comment:
The file is called `get_esql_client.ts` but it exports `getEsClient`. Is that intentional?

Reply:
Good catch. I forgot to rename it